core/amber/src/main/python/core/models/payload.py

```diff
@@ -1,25 +1,16 @@
 from dataclasses import dataclass
 from pyarrow.lib import Table
-from typing import List, Optional
-
-from core.models.schema.schema import Schema
-from core.models.tuple import Tuple
+from typing import Optional
 
 
 @dataclass
 class DataPayload:
-    pass
-
-
-@dataclass
-class InputDataFrame(DataPayload):
-    frame: Table
+    frame: Optional[Table] = None
 
 
 @dataclass
-class OutputDataFrame(DataPayload):
-    frame: List[Tuple]
-    schema: Optional[Schema] = None
+class DataFrame(DataPayload):
+    pass
```

Comment:
I see you are simplifying this wrapper design, which is a good thing. A few things to notice:
1. why does `DataPayload` has an optional Table called `frame`? isn't frame a type of data payload? this is similar to define 
```
class Animal:
        dog: Optional[Object]

class Dog(Animal):
        pass
```
which is weird. I'd suggest 
```
@dataclass
class DataPayload:
    data

@dataclass
class DataFrame(DataPayload):
    data: Table

@dataclass
Class MarkerFrame(DataPayload):
    data: Marker
```
2. Can you make sure the change is aligned with java side?